tests/end-to-end/upgrade/chart/upgrade/values.yaml

```diff
@@ -3,7 +3,7 @@ containerRegistry:
 
 image:
   dir:
-  version: 55ccf207
+  version: PR-7755
```

Comment:
Is this for verification purposes?

As this change should be in another PR with the final image SHA.

Reply:
yes that will happen ... but as all those updated tests create new images i want to wait until all evidence of event-bus has been wiped from the test folder and then create one pr to update all of them